sass/typography/_functions.scss

```diff
@@ -78,10 +96,10 @@
 /// @access public
 /// @param {String} $font-family [var(--ig-font-family)] - The font family of the type style.
 /// @param {String} $font-size - The font size of the type style.
-/// @param {Number|String} $font-weight [normal] - The font weight of the type style.
+/// @param {Number} $font-weight [normal] - The font weight of the type style.
```

Comment:
You removed the `String` as possible type to use for the font-weight, but the default value is `normal`, not `400`.  Also if it's valid to pass `bold`, `semi-bold`, etc. I think `String` is still valid?